stories/patterns/filters/add-filter/examples/docs/add-filter.tsx

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import AddFilterPattern from '@semcore/add-filter-pattern';
+import Select from '@semcore/select';
+import Button from '@semcore/button';
+import { Flex } from '@semcore/flex-box';
+
+import SearchM from '@semcore/icon/Search/m';
+import { ButtonLink } from '@semcore/button';
+import CloseM from '@semcore/icon/Close/m';
+import { Hint } from '@semcore/tooltip';
+import { Text } from '@semcore/typography';
+import Radio, { RadioGroup } from '@semcore/radio';
+import Textarea from '@semcore/textarea';
+import type { AddFilterPatternDropdownProps } from '@semcore/add-filter-pattern';
+import { verify } from 'crypto';
```

Comment:
you don't use this import and on the 14th line

Reply:
removed